intellst-api/src/Controller/AuthController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Service\Register;
+
+class AuthController extends AbstractController
+{
+    /**
+     * AuthController constructor.
+     * @param Register $register
+     */
+
+    public function __construct( Register $register ) {
```

Comment:
please enable PSR-12 styling in PHP storm to avoid comment with spacing like here:
remove spaces around `Register $register`

Reply:
enabled 
![Screenshot from 2020-06-25 12-04-03](https://user-images.githubusercontent.com/45236851/85690275-18999480-b6dc-11ea-8395-50855added37.png)